### ISSUE TITLE
Decide whether to continue to create props to get imported shadcn classes to pass eslint or correctly configure eslint to ignore that problem in those files.

### ISSUE THREAD

**Issue by tombogle** [paranext/paranext-core#939]

**Comment by irahopkinson** [comment]
I was just watching the UX+Dev meeting recording where this was talked about from a couple weeks back. I wouldn't bother to reconfigure `eslint` for these ShadCN copied components. Simply add an ignore-this-file-comment to the top of any file when copied in. e.g. `/* eslint-disable */`. That way the file can be copied as is and not modified to fit our system.

**Comment by tombogle** [comment]
I believe it is relatively easy (at least it _should_ be) to configure eslint to not report certain errors for any files in that folder. Probably less of a hassle in the long run than having to add a disable comment into each file. (Might already be done now.)

**Comment by irahopkinson** [comment]
Right, if they all in one folder you can add that folder to the `.eslintignore` file. That would be a better solution.
